### ISSUE TITLE
Login page missing labels

### ISSUE THREAD

**Issue by gnomeontherun** [clr-team/planning#7]
## Description of work

Th form on login page was missing labels, and need them for accessibility.

## Issue(s)

Fixes https://github.com/vmware/clarity/issues/3522

## Reviewers

* [ ] Bozhidar
* [ ] Cory
* [ ] Ivan
* [ ] Jeeyun
* [ ] Jeremy
* [x] Matt
* [x] Scott
* [ ] Shijir

## Definition of done

* [ ] Tests completed
* [ ] Reviewed by peers
* [ ] Jehad review
* [ ] Design sign off (if necessary)
* [ ] Merged into master/topic branch
